src/components/Editor/Preview/Popup.js

```diff
@@ -34,6 +34,10 @@ type Props = {
   openLink: string => void
 };
 
+function isReactComponent(roots) {
+  return roots.some(root => root.name === "_reactInternalInstance");
```

Comment:
haha - we'll get back to this... as we do other objects

and yeah, i kinda like looking at a super secret internal prop